src/test/ui/parser/issue-90728.stderr

```diff
@@ -0,0 +1,20 @@
+error: expected at least one digit in exponent
+  --> $DIR/issue-90728.rs:2:7
+   |
+LL |     a.5.2E+
+   |       ^^^^^
```

Comment:
This error in particular could likely be more actionable.

Reply:
Could you be a bit more precise about that?

Reply:
I mean using a `span_suggestion_verbose` to produce something like the following (just a rough draft, doesn't have to look like this exactly):
```
error: expected at least one digit in exponent
  --> $DIR/issue-90728.rs:2:7
   |
LL |     a.5.2E+
   |       -----^ expected a number here
   |       |
   |       this numeric literal is incomplete
help: write a valid exponent
   |
LL |     a.5.2E+1
   |            +